netlify.toml

```diff
@@ -39,3 +39,8 @@
   from = "/helm/v2/*"
   to = "/helm/v2"
   status = 200
+
+[[redirects]]
+  from = "/calendar/release"
```

Comment:
I wondered if the trailing slash mattered here, but this looks good per https://docs.netlify.com/routing/redirects/